openqabot/openqa.py

```diff
@@ -74,7 +74,9 @@ def get_job_comments(self, job_id: int):
             )
             ret = list(map(lambda c: {"text": c.get("text", "")}, ret))
         except Exception as e:
-            logger.exception(e)
+            (method, url, status_code) = e.args
+            if status_code != 404:
+                logger.exception(e)
```

Comment:
Maybe it could still be helpful to at least log that a job was not found.
The exception was too noisy, but a one-line info message?

Reply:
I don't know. Let's see what the rest thinks.

Reply:
Ok, agreed